src/containers/TrendContainer.js

```diff
@@ -80,31 +81,30 @@ class TrendContainer extends React.Component {
 
   render() {
     const {
-      crime,
-      place,
       places,
-      placeType,
-      since,
       summaries,
-      until,
+      region,
+      states,
+      filters,
     } = this.props
+    const placeName = lookupDisplayName(filters, region.regions, states.states)
     const isReady = !summaries.loading
 
     let otherCrimes = []
-    if (crime === 'violent-crime') {
+    if (filters.crime === 'violent-crime') {
       otherCrimes = ['homicide', 'rape', 'robbery', 'aggravated-assault']
-    } else if (crime === 'property-crime') {
+    } else if (filters.crime === 'property-crime') {
       otherCrimes = ['arson', 'burglary', 'larceny', 'motor-vehicle-theft']
     }
-
+    const crime = filters.crime
     return (
       <div className="mb7">
         <div className="mb2 p2 sm-p4 bg-white border-top border-blue border-w8">
           <h2 className="mt0 mb2 sm-mb4 fs-24 sm-fs-28 sans-serif">
-            {startCase(crime)} rate in {lookupUsa(place).display}, {since}-{until}
+            {startCase(filters.crime)} rate in {placeName}, {filters.since}-{filters.until}
```

Comment:
Place name here results in some weird stuff for regions. For example, "Robbery rate in West Region, 2006-2016" when it probably should be "Robbery rate in the West region, 2006-2016" since "West Region" is neither a proper noun